styles/style.css

```diff
@@ -529,7 +529,7 @@ form textarea {
   border-radius: 10px;
 }
 
-@media screen and (min-width: 995px) {
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] (Optional) It would be nice to center the `button` with class `get-it-touch-btn` so that your design for the form looks like that of the Figma template you chose.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/160216701-46f5bf3e-324c-4c01-b3ad-6963884b5f70.png)


### Figma Template

![image](https://user-images.githubusercontent.com/77106962/160216731-1a3601cc-df97-4df3-8911-fb086c016140.png)

Reply:
@Gambit142 Thank you for the feedback